### DISCUSSION TITLE
啥原因

### DISCUSSION THREAD

**Discussion by cbw7172002** [doocs/md#242] [General] [answered]
code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '0.0.0.0',
  port: 9000
} uncaughtException
40% building 9/15 modules 6 active ...ifan\Documents\git\md\node_modules\webpack-dev-server\client\utils\sendMessage.jsexit null
exit: null
98% after emitting CopyPlugin

 ERROR  Failed to compile with 2 errors                                                                         13:36:42

These dependencies were not found:

* core-js/modules/web.url-search-params.delete.js in ./src/assets/scripts/util.js
* core-js/modules/web.url-search-params.has.js in ./src/assets/scripts/util.js

**Comment by yanglbme** [comment, marked as answer]
项目已升级至 vue3
